### ISSUE TITLE
Unable to customize aria-label for Nav

### ISSUE THREAD

**Issue by seanforyou23** [patternfly/patternfly-react#4507]
**Describe the issue. What is the expected and unexpected behavior?**

When I set `aria-label="Foo"` on the [Nav](https://www.patternfly.org/v4/documentation/react/components/nav#default) component, it applies the attribute as `aria-label="Local"`. As a result, the custom aria-labels we set in org are not coming through. Is it something we can fix? Does this prop need to be added to the NavContext by chance?

**Please provide the steps to reproduce. Feel free to link CodeSandbox or another tool.**
https://www.patternfly.org/v4/documentation/react/components/nav#default

**Is this a bug or enhancement? If this issue is a bug, is this issue blocking you or is there a work-around?**

Bug

**Comment by jessiehuff** [comment]
I agree that this is something we should try to fix. Not only does it mean that we can't customize our own documentation, it likely means that consumers would be limited to "Global" and "Local" as aria-labels unless they created their own workaround.
